pkg/apis/apiserver/organization_REST.go

```diff
@@ -361,12 +409,7 @@ func (o *OrganizationREST) isVisible(ctx context.Context, organization *Organiza
 		return true, nil
```

Comment:
Why returns true here?

Reply:
because the only way that's happening if you're disabling auth in the API exstension server via `--delegate-auth=false`

Reply:
Ok, but why in this case, we expose all organizations objects in bulward to unknown user?

Reply:
because this is only for debugging, you do not run API extension under normal circumstances.